src/Controls/src/Core/Element.cs

```diff
@@ -128,9 +129,9 @@ internal IEnumerable<Element> AllChildren
 
 		internal virtual IEnumerable<Element> ChildrenNotDrawnByThisElement => EmptyChildren;
 
+		IReadOnlyList<Element> IElementController.LogicalChildren => LogicalChildrenInternal;
 
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public ReadOnlyCollection<Element> LogicalChildren => LogicalChildrenInternal;
+		internal IReadOnlyList<Element> LogicalChildren => LogicalChildrenInternal;
```

Comment:
This is now internal (optionally accessible via the `IElementController` interface)

This will be replaced in future with `IVisualTreeElement` (which is merged and working already)